.github/workflows/ci.yml

```diff
@@ -43,8 +43,8 @@ jobs:
           deps-${{ runner.os }}-${{ matrix.otp }}-${{ matrix.elixir }}
     - name: Install dependencies
       run: mix deps.get
-    - name: Check code is formatted
-      run: mix format --check-formatted
+        #    - name: Check code is formatted
+        #      run: mix format --check-formatted
```

Comment:
@SimonLab why are we skipping `mix format` on `CI`? 🤷‍♂️ 
Is there an issue where this is captured? 💭

Reply:
Linked issue: https://github.com/dwyl/mvp/issues/183

I found the "format check" on the CI annoying. `mix format` was adding new line with the `<textarea>` tags which created tasks with empty spaces at the start. I've resolved this by adding a content to the textarea which seem to pleased the formatter.
I'll uncomment the lines but I'm still not sure it's a good idea to have this check, a more flexible approach would be to run the command manually; undecided